rules/common_settings.bzl

```diff
@@ -123,6 +123,24 @@ string_list_flag = rule(
     doc = "A string list-typed build setting that can be set on the command line",
 )
 
+def _flag_impl(ctx):
+    return BuildSettingInfo(value = ctx.build_setting_value)
+
+repeatable_string_flag = rule(
+    implementation = _flag_impl,
+    build_setting = config.string_list(
+        flag = True,
+        repeatable = True,
+    ),
+    attrs = {
+        "scope": attr.string(
+            doc = "The scope indicates where a flag can propagate to",
+            default = "universal",
+        ),
```

Comment:
Pre-existing issue: we should move scope to a variable (like `_MAKE_VARIABLE_ATTR`) to avoid so much repetition...

Can be done in follow-up